src/optimal_image/dataclients/mod.rs

```diff
@@ -1,14 +1,18 @@
 extern crate dssim;
+extern crate rgb;
 use imgref::ImgVec;
+use rgb::*;
 use std::error::Error;
 use std::path::Path;
+mod jpeg;
 mod png;
 
-pub type ImageData = ImgVec<dssim::RGBAPLU>;
-pub type ImageDataResult<E = Error> = Result<ImageData, E>;
+pub type ImageData = ImgVec<RGBA<f32>>;
```

Comment:
Would this work for PNG then?

Reply:
yeah, that's just an alias.
dssim requires that format to the math